app/models/miq_product_feature.rb

```diff
@@ -1,8 +1,10 @@
 class MiqProductFeature < ApplicationRecord
-  SUPER_ADMIN_FEATURE = "everything".freeze
+  SUPER_ADMIN_FEATURE   = "everything".freeze
   REPORT_ADMIN_FEATURE  = "miq_report_superadmin".freeze
   REQUEST_ADMIN_FEATURE = "miq_request_approval".freeze
-  TENANT_ADMIN_FEATURE = "rbac_tenant".freeze
+  REPORT_MY_TASKS       = "miq_task_my_ui".freeze
```

Comment:
Following the existing convention I think these two should be named with `*_FEATURE`.

Also, the `REPORT` prefix is misleading because these features are not reporting feature, afaik. Perhaps you can name them `MY_TASKS_FEATURE` and `ALL_TASKS_FEATURE`?

Reply:
👍 Thank you. Will do